R/utils.R

```diff
@@ -74,3 +74,113 @@ write_test <- function(
     , file
   )
 }
+
+
+
+
+new_test_data <- function(numrow = 10){
```

Comment:
Maybe we should add an argument to set the seed?  That way we can get identical results when needed. Something like: `function(numrow = 10, seed = 100)`